apps/user-profile/src/app/pages/profile-page/profile-page.component.ts

```diff
@@ -120,20 +120,20 @@ export class ProfilePageComponent implements OnInit {
         const emailAttribute = richUser.userAttributes.find(
           (att) => att.friendlyName === 'preferredMail'
         );
-        this.email = emailAttribute.value as unknown as string;
+        this.email = emailAttribute.value as string;
```

Comment:
also Mato wanted to cover the case when the email I null/undefined (can happen)
```suggestion
        this.email = emailAttribute?.value ? emailAttribute?.value  as string : '-';
```